docs/source/_templates/partials/_head_css_variables.html

```diff
@@ -0,0 +1,9 @@
+<!-- DO NOT REMOVE THIS FILE
+
+This file is normally inherited from the version defined by our docs theme (Furo).
+However, that file includes in-line styling which isn't allowed by our website's
+Content Security Policy (CSP). To resolve this issue, we've copied over the styling changes
+to `docs/source/_static/css/dark_light_mode.css`. Although this file is empty, it overwrites
+the template inherited from Furo. 
+
+ -->
```

Comment:
We should make this a jinja comment and not html comment as shown in the example below:

```
{#
  DO NOT REMOVE THIS FILE

  This file is normally inherited from the version defined by our docs theme (Furo).
  However, that file includes in-line styling which isn't allowed by our website's
  Content Security Policy (CSP). To resolve this issue, we've copied over the styling changes
  to `docs/source/_static/css/dark_light_mode.css`. Although this file is empty, it overwrites
  the template inherited from Furo. 
#}
```
This will prevent this comment from being rendered in every html page we generate